src/components/fileviewer.js

```diff
@@ -224,17 +224,19 @@ const Line = ({ lineNumber, lineText, coverage, selectedLine, onLineClick }) =>
 };
 
 /* This component contains metadata of the file */
-const FileViewerMeta = ({ revision, path, status, coverage }) => {
+const FileViewerMeta = ({ revision, path, self, coverage }) => {
```

Comment:
..as per last comment; status will not exist at this point, use `self` (an alias of `this`)